addon-test-support/setup-mirage-test.js

```diff
@@ -0,0 +1,24 @@
+import startMirage from 'ember-cli-mirage/start-mirage';
+
+//
+// Used to set up mirage for a test. Must be called after one of the
+// `ember-qunit` `setup*Test()` methods. It starts the server and sets
+// `this.server` to point to it, and shuts the server down when the test
+// finishes.
+//
+export default function setupMirageTest(hooks) {
```

Comment:
Do folks have any strong opinions about `setupMirageTest(hooks)` vs just `setupMirage(hooks)`?

Based on the way e.g. `setupRenderingTest(hooks)` implicitly sets up the test context so you don't _also_ need to call `setupTest(hooks)`, I think I would expect a function called `setupMirageTest` to do the same.

For me, `setupMirage` on its own doesn't carry that same implication, so might be clearer.

Reply:
If I had to vote, I'd also prefer `setupMirageTest`

Reply:
You get a vote! And you get a vote! Everybody gets a vote!

Reply:
👍  for setupMirageTest if I get a vote as well ;-)

Reply:
After thinking about this some more, I think I also prefer `setupMirage` over `setupMirageTest` since it distinguishes it from the methods that set up the test (context) itself, and also the word `test` is totally redundant since this method is only available in tests (imported from `addon-test-support/`), so I don't see that it adds anything.

Does anybody have an argument for `setupMirageTest` other than vague personal preference?